src/form/Checkbox/CheckboxTheme.ts

```diff
@@ -5,6 +5,11 @@ export interface CheckboxTheme {
   label: {
     base: string;
     clickable: string;
+    sizes: {
```

Comment:
Need to update docs

Reply:
It's already updated.
That using ThemeRender which using variable
`<ThemeRender theme={darkCheckboxTheme} />`

![Screenshot 2024-03-26 at 15 38 54](https://github.com/reaviz/reablocks/assets/5399389/7ee222d1-8b8e-4f3d-8de7-3998f20635b1)